packages/front-generator/src/generators/react-typescript/app/template/src/config.ts

```diff
@@ -1,3 +1,5 @@
 export const CUBA_APP_URL = process.env.REACT_APP_CUBA_URL || '/app/rest/';
+export const REST_CLIENT_ID = process.env.REACT_APP_REST_CLIENT_ID || "client";
+export const REST_CLIENT_SECRET = process.env.REACT_APP_REST_CLIENT_SECRET || "secret";
```

Comment:
A nitpick, but since we are using TypeScript 3.7+, we can use `??` null coalescing operator.